### ISSUE TITLE
problem when you request (delete_payment) arabic language !

### ISSUE THREAD

**Issue by manlive** [opensourcepos/opensourcepos#2261]
PHP 5.6 MYSQL 5.7 APACHE 2.4

hello , 
there problem when i press (delete_payment) on sales page 

```
Object not found!
The requested URL was not found on this server.

```

< pic : 

![](https://i.imgur.com/7ZMWq5V.jpg)

thank you 
(ps : sorry for asking this but i see a lot of good commits and fixes up , can you make new release soon or maybe 'll w8 for new V CodeIgniter ?)
thank you again

**Comment by SteveIreland** [comment]
What version of OSPOS are you working with here?

**Comment by manlive** [comment]
> 
> 
> What version of OSPOS are you working with here?
OSPOS 3.2.3,

is this bug (when i delete (items) from items page , items still in database , how i remove items completely from items page

**Comment by odiea** [comment]
When you delete an Item it is soft deleted so that it will still appear in receipt reprints at a later date.

**Comment by manlive** [comment]
this problem fixed on 3.3 , 
i'll try fixed some (text values in Arabic language) i see some bugs
